index.html

```diff
@@ -2804,8 +2818,9 @@ <h3><dfn>Writing content</dfn></h3>
 
     <section><h3>Mapping empty record to NDEF</h3>
       <div>
-        To <dfn>map empty record to NDEF</dfn> given a |record:NDEFRecordInit|
-        |ndef| and |context:string|, run these steps:
+        To <dfn>map empty record to NDEF</dfn> given |record:NDEFRecordInit|,
+        |ndef|, |context:string|, and |recordsDepth:unsigned short|,
```

Comment:
Can we skip passing the depth to algorithms like this which don't use it?

Reply:
I've done this for `recordsDepth` and `context`.